pkg/commands/execution_context.go

```diff
@@ -255,15 +257,16 @@ func importFiles(testConfig *config.CloudTestConfig, files ...string) error {
 // PerformTesting performs testing uses cloud test config. Returns the junit report when testing finished.
 func PerformTesting(config *config.CloudTestConfig, factory k8s.ValidationFactory, arguments *Arguments) (*reporting.JUnitFile, error) {
 	ctx := &executionContext{
-		cloudTestConfig:  config,
-		operationChannel: make(chan operationEvent, 100),
-		tasks:            []*testTask{},
-		running:          map[string]*testTask{},
-		completed:        []*testTask{},
-		tests:            []*model.TestEntry{},
-		factory:          factory,
-		arguments:        arguments,
-		manager:          execmanager.NewExecutionManager(config.ConfigRoot),
+		cloudTestConfig:    config,
+		operationChannel:   make(chan operationEvent, 100),
+		terminationChannel: make(chan error, 100),
```

Comment:
Do we really need set size 100 for the buffer of channel `terminationChannel` ?

Reply:
See [my another comment](https://github.com/networkservicemesh/cloudtest/pull/25#discussion_r386839042). I have an idea to set it to `max(10,len(CloudTestConfig.HealthChecks))`